### ISSUE TITLE
Experiment with yield diff map concept

### ISSUE THREAD

**Issue by kelsey-taylor** [earthrise-media/vacs-map#16]
blocked by #2 and #13

**Comment by kelsey-taylor** [comment]
the current designs don't feature this concept per se but we talked about ways to add it back yesterday...moving to backlog to focus on other visuals

**Comment by kelsey-taylor** [comment]
gonna close this as no longer relevant
